pyCFOFiSAX/_tree_iSAX.py

```diff
@@ -264,29 +264,29 @@ def insert(self, new_sequence):
 
     def insert_paa(self, new_paa):
         """
-        La fonction insert qui appelle directement celle de son nœud root
+        The insert function that directly calls that of its root node
 
-        :param numpy.array new_paa: la nouvelle séquence à insérer
+        :param numpy.array new_paa: The new sequence to be inserted
         """
 
         if len(new_paa) < self.size_word:
-            print("Erreur !! "+new_paa+" est plus petit que size.word = "+self.size_word+". FIN")
+            print("Error !! "+new_paa+" is smaller than size.word = "+self.size_word+". FIN")
         else:
             self.root.insert_paa(new_paa)
             self._new_insertion_after_preproc = True
             self._new_insertion_after_minmax_nodes = True
 
     def preprocessing_for_icfof(self, ntss_tmp, bool_print: bool = False, count_num_node: bool = False):
         """
-        Permet de faire appel, pour l'arbre ``id_tree`` aux deux méthodes de pré-traitements:
+        Allows us to appeal, for the ``id_tree`` tree to the two methods of pre-treatments:
```

Comment:
Allows us to appeal, for the `id_tree` tree, to the two methods of preprocessing

Reply:
```suggestion
        Allows us to appeal, for the id_tree tree, to the two methods of preprocessing:
```